### ISSUE TITLE
enhancement: Allow Fuzzy Skin to be printed on a face

### ISSUE THREAD

**Issue by ipodsync** [bambulab/BambuStudio#637]
In addition to being able to apply fuzzy skin to outside or all faces it would be nice to be able to specify the faces to apply it to.  

I'd love to leverage this ability, but the present options make impractical for what I'm printing.

**Comment by Benderrodrigu3z** [comment]
there is a paint bucket tool, is that what your looking for?

**Comment by JstBecause** [comment]
> there is a paint bucket tool, is that what your looking for?

A paint bucket tool, or brush, etc to choose which surfaces you want the fuzzy skin on would be a great addition.  I have a print I want to try just a few walls to be fuzzy, while keeping the inside (due to how tightly it fits around another piece) not fuzzy.

**Comment by psiberfunk** [comment]
It feels like you can almost do it with the present options, but the modifier objects do stupid things like make extra lines.  Here's an example with a cylinder and a modifier cube, which makes only part of it fuzzy.. but introduces a new obnoxious set of walls in the middle of the cylinder.. to be fair, this is also a problem in upstream prusaSlicer.

![image](https://user-images.githubusercontent.com/19777571/202324728-93775bc7-c9cc-42a6-a0eb-8caedde599ee.png)

Results in:

![image](https://user-images.githubusercontent.com/19777571/202324762-c0f5e6e6-86c5-4185-8530-e6b1f3c72cb8.png)

**Comment by psiberfunk** [comment]
Interestingly, in prusaSlicer, you can do this with with a good effect with height range modifiers (but probably not what you want here).. which BambuSlicer sorely lacks.. I wish they would have it for other reasons:

![image](https://user-images.githubusercontent.com/19777571/202325093-868bcd59-5aab-4c80-b444-c51a0e9c1801.png)

**Comment by JstBecause** [comment]
> It feels like you can almost do it with the present options, but the modifier objects do stupid things like make extra lines.  Here's an example with a cylinder and a modifier cube, which makes only part of it fuzzy.. but introduces a new obnoxious set of walls in the middle of the cylinder.. to be fair, this is also a problem in upstream prusaSlicer.
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/19777571/202324728-93775bc7-c9cc-42a6-a0eb-8caedde599ee.png)
> 
> 
> 
> Results in:
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/19777571/202324762-c0f5e6e6-86c5-4185-8530-e6b1f3c72cb8.png)
> 
> 
> 
> 
> 
> 
> 
> 

Yeah this is a known bug in all these products. Or maybe not be a bug, but not how it is designed. Would work absolutely great if it did work the way it is supposed to! Would make a lot of peoples lives easier. 🤣

**Comment by Benderrodrigu3z** [comment]
if you want to all be fuzzy then put it over the entire object? not sure why that's difficult.... again I don't work for Bambu

**Comment by psiberfunk** [comment]
> if you want to all be fuzzy then put it over the entire object? not sure why that's difficult.... again I don't work for Bambu

That’s not what he’s asking for .  He’s asking for the ability to “paint” the fuzzy ness , which is a good idea .

**Comment by SillacSaurfang** [comment]
I would absolutely love this, I've always wanted it from a slicer. Fuzzy skin as it works now is great for single-piece items, but manually blocking out the internals or interfacing parts of a complex or curved object with modifiers can be a nightmare.

**Comment by Photogad** [comment]
Yes, PLEASE!

**Comment by hutje** [comment]
Apparently this problem is still not fixed😔
Is there any possibility it would be released anytime soon?

**Comment by magnusmyklebost** [comment]
Prusa have done it. Why cant bambu? 
https://github.com/prusa3d/PrusaSlicer/releases/tag/version_2.9.0-alpha1

**Comment by hutje** [comment]
> Prusa have done it. Why cant bambu? https://github.com/prusa3d/PrusaSlicer/releases/tag/version_2.9.0-alpha1

Because Prusa has just released it in their alpha version. Give it time. Most new features are getting to all open source slicers eventually. Most likely OrcaSlicer will implement it first, then BambuStudio...

**Comment by hutje** [comment]
I also hope it will eventually be possible to have the option to load a depth map image with a texture to emboss this in the surface. Now this process has a lot of steps by using Blender or specific 3D-cad software to emboss the object.
But this way you can create leather-like textures or woodgrains on an object.
![1000109142](https://github.com/user-attachments/assets/94e07a96-9174-44fc-ac1b-11b48a46b411)

**Comment by magnusmyklebost** [comment]
Very cool, but i think fuzzy skin feature or other grip textures will be more usefull as this will help functional prints with gripping friction as well as being used for cosmetic reasons.

**Comment by alie2n** [comment]
I just stumbled accross this issue and would also second this thought. Adding arbitrary fuzzy skin would be great.
